auth_options.go

```diff
@@ -182,10 +182,45 @@ func (opts *AuthOptions) ToTokenV3CreateMap(scope map[string]interface{}) (map[s
 		Auth authReq `json:"auth"`
 	}
 
+	appCredReq := func(opts *AuthOptions) (*userReq, error) {
```

Comment:
Let's put all of these conditionals inline with the rest of the logic below. Admittedly it's a long string of if/else statements and the whole thing could probably be refactored, but let's stick with consistency.

Reply:
Done